Formula/gnuradio.rb

```diff
@@ -6,7 +6,7 @@ class Gnuradio < Formula
   url "https://github.com/gnuradio/gnuradio/archive/refs/tags/v3.10.6.0.tar.gz"
   sha256 "c1a8c09f975d224b27dad9c8e2e7fed1f7cde9bc04d30b004afe8e28bb3f45c6"
   license "GPL-3.0-or-later"
-  revision 1
+  revision 2
```

Comment:
Let's not force everyone to reinstall this just to change a build dependency, please.
```suggestion
  revision 1
```